### ISSUE TITLE
Support NestJs 9

### ISSUE THREAD

**Issue by adrian-goe** [whefter/nestjs-joi#20]
Hey,
we tried to update to nestJs 9 and got an error

```
npm ERR! Found: @nestjs/common@9.0.11
npm ERR! node_modules/@nestjs/common
npm ERR!   @nestjs/common@"9.0.11" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @nestjs/common@"^7.0.0 || ^8.0.0" from nestjs-joi@1.8.0
npm ERR! node_modules/nestjs-joi
npm ERR!   nestjs-joi@"1.8.0" from the root project
```

Could you update the dependencies?

**Comment by slaveofcode** [comment]
Please update, this block for nest v9

**Comment by whefter** [comment]
I've released v1.8.1, which adds NestJS v9 as a peerDependency. Note that installing older versions should be possible by setting `--legacy-peer-deps=true`.

According to my tests, there are no actual functional conflicts by upgrading. Let me know if you run into any.

**Comment by whefter** [comment]
Edit: there seems to be a problem with the package publishing, will review later.

**Comment by whefter** [comment]
Edit: this was fixed in 1.8.1, forgot to close.
